components/automate-ui/src/app/components/resource-dropdown/resource-dropdown.component.ts

```diff
@@ -100,14 +110,33 @@ export class ResourceDropdownComponent implements OnInit, OnChanges {
 
   closeDropdown(): void {
     this.dropdownState = 'closed';
-    this.onDropdownClosing.emit(
-      this.resources.filter(r => r.checked).map(r => r.id));
+    const flattenedIDs = [].concat(
+      ...this.snapshotResources.map(
+        resource => resource.itemList.filter(r => r.checked).map(r => r.id)));
```

Comment:
Huh that's an interesting construct. What's the benefit of `[].concat(...this.snapshotResources.map(/* ... */)` over `this.snapshotResources.flatMap(/* ... */)`? (⚠️ untested)

Reply:
None; just did not come to mind. Thanks!